tracks/julia/exercises/word-count/mentoring.md

```diff
@@ -27,10 +27,10 @@ Dict{String,Int64} with 6 entries:
 
 **Split, then strip quotation**
 
-This solution, like all the others below uses "Regex".
+This solution, like all the others below uses "regex".
 Regex is a concise language for describing patterns in strings.
```

Comment:
```suggestion
Regex (short for regular expression) is a concise language for describing patterns in strings.
```

Reply:
I think it is best not to expand this because:

1. "Regular expression" is a name, but doesn't help you understand the concept at all
2. PCRE regexes aren't regular expressions